build.gradle

```diff
@@ -255,23 +256,27 @@ node {
 }
 
 task configureKoop {
-    copy {
-        from("src/koop/config") {
-            include "**"
-            filter(org.apache.tools.ant.filters.ReplaceTokens, tokens: [
-                KOOP_PORT : project.property("koopPort"),
-                KOOP_SSL_ENABLED : project.property("koopSSLEnabled"),
-                KOOP_SSL_PORT : project.property("koopSSLPort"),
-                KOOP_SSL_CERT : project.property("koopSSLCert"),
-                KOOP_SSL_KEY : project.property("koopSSLKey"),
-                ML_HOST : mlAppConfig.host,
-                ML_PORT : mlAppConfig.restPort.toString(),
-                ML_USER : mlAppConfig.restAdminUsername,
-                ML_PASSWORD : mlAppConfig.restAdminPassword
-            ])
-        }
+    outputs.upToDateWhen { false }
```

Comment:
Does this make it overwrite no matter what? I added the "delete file" line below to force it to update `default.json` even if the file hasn't changed on disk but the replace tokens have been changed.

Reply:
The outputs.upToDateWhen { false } tells Gradle not to do an incremental build check.